runtime/sema/type.go

```diff
@@ -3354,6 +3389,35 @@ func (t *CompositeType) IsStorable(results map[*Member]bool) bool {
 	return true
 }
 
+func (t *CompositeType) IsImportable(results map[*Member]bool) bool {
+	// Use the pre-determined flag for native types
+	if t.Location == nil {
+		return t.importable
+	}
+
+	// Only structures, resources, and enums can be imported
+
+	switch t.Kind {
+	case common.CompositeKindStructure,
+		common.CompositeKindResource,
```

Comment:
Resources may not be imported!

Reply:
I kept that as is, because it was also the current behavior. But yeah makes sense to restrict it.

Added with 9381434